docs/en/api/config.md

```diff
@@ -79,3 +79,18 @@ VueTestUtils.config.provide['$logger'] = {
   }
 }
 ```
+
+### `logModifiedComponents`
+
+- type: `Boolean`
+- default: `true`
+
+Logs warning when extended child components are automatically stubbed. Hides warnings when set to `false`.
```

Comment:
I think that it is better to describe that `logModifiedComponents` can not be overwritten by mounting options.